### ISSUE TITLE
cucumberOpts.require: ['./stepsFolder'] produces module not found error

### ISSUE THREAD

**Issue by FlippieCoetser** [webdriverio/wdio-cucumber-framework#79]
```Javascript
cucumberOpts: {
    require: ['./stepsFolder']
}
```
When using the folder of the steps in the require section, and not the individual files, an error is produced: module not found.

Workaround solution: Add a helper at the top of the wdio.conf.js file

```Javascript
const stepsFolder = './stepsFolder';
const fs = require('fs');

var steps = []

fs.readdirSync(stepsFolder).forEach(file => {
    step = stepsFolder + '/' + file;
    steps.push(step);
  });
```
 With config

```Javascript
cucumberOpts: {
    require: steps
}
```

**Comment by christian-bromann** [comment]
@FlippieCoetser this is by design (from cucumber.js). I would love to have support for globbing here and I am sure this can be implemented.

PRs welcome!

**Comment by FlippieCoetser** [comment]
@christian-bromann, before I start helping develop a more permanent solution, here is the BDD spec to confirm the **current** and **expected** behavior:

**Background:**

Given `package.json` file exist
And `package.json` file contains devDependencies `"webdriverio": "^4.8.0"`
And `package.json` file contains script `"test": "wdio"` 
And `wdio.conf.js` file does not exist
When I execute `npm test` in command line
Then the `WDIO Configuration Helper` should start

Given `WDIO Configuration Helper` started
And `One my local machine` was selected for `Where do you want to execute your tests?`
And `cucumber` was selected for `Which framework do you want to use?`
And `Y` was entered for `Shall I install the framework adapter for you?`
And the enter key was pressed to accept the default `./features/**/*.feature` for `Where are your feature files located?`
And the enter key was pressed to accept the default **`./features/step-definitions`** for `Where are your step definitions located?`
And `spec` was selected for `Which reporter do you want to use?`
And `Y` was entered for `Shall I install the reporter library for you?`
And `selenium-standalone` was selected for `Do you want to add a service to your test setup?`
And `Y` was entered for `Shall I install the services for you?`
And `silent` was selected for `Level of logging verbosity`
And the enter key was pressed to accept the default `./errorShots/` for `In which directory should screenshots gets saved if a command fails?`
And the enter key was pressed to accept the default `http://localhost` for `What is the base url?`
Then a `wdio.conf.js` file should exist
And the `wdio.conf.js` file should contain **`cucumberOpts.require: ['./features/step-definitions']`**

**Current:**

Given the `wdio.confs.js` file contains **`cucumberOpts.require: ['./features/step-definitions']`**
And a `step.js` file is located in the folder './features/step-definitions'
When I execute `npm test` in command line
Then an `ERROR: Cannot find module` is produced
And the `step.js` is not loaded 

**Expected:**

Given the `wdio.confs.js` file contains **`cucumberOpts.require: ['./features/step-definitions']`**
And a `step.js` file is located in the folder './features/step-definitions'
When I execute `npm test` in command line
Then `step.js` should be loaded
And the relevant tests should be executed

**Comment by walternolasco** [comment]
Hello guys,
Can somebody please help me on the following:
I'm getting this error with the environment:
"devDependencies": {
    "cucumber": "^3.1.0",
    "wdio-cucumber-framework": "^1.0.2",
    "wdio-selenium-standalone-service": "0.0.9",
    "webdriverio": "^4.9.8"
  }

I've tried setting up the step definitions directory without globbing and I continue receiving the './stepsFolder'] produces module not found error.

Is there any workaround to make it work for the moment?
Thanks in advance for your valuable help!

**Comment by FlippieCoetser** [comment]
@walternolasco you need to name the complete filename of the step file: 

```
cucumberOpts: {
    require: ['./stepsFolder/stepfile.js']
}
```
As a workaround to load several stepsfiles in a specific folder, I created a helper function:

```
const stepsFolder = './stepsFolder';
const fs = require('fs');

var steps = []

fs.readdirSync(stepsFolder).forEach(file => {
    step = stepsFolder + '/' + file;
    steps.push(step);
  });
```
which then allowed me to just add the newly generated array of filenames by just pointing to the array name.
```
cucumberOpts: {
    require: steps
}
```

**Comment by walternolasco** [comment]
Thank you very much @FlippieCoetser Got it working now.  👍

**Comment by nicangeli** [comment]
I think this is closed by #90?

**Comment by christian-bromann** [comment]
thanks @nicangeli
